website/docs/basics/blocks-component-in-block.md

```diff
@@ -728,7 +728,7 @@ You can also do this and this is why we created the third parameter in the props
 
 typographyContent attribute will follow the naming prefix when passed to a child but the disableEdit will remain the same name in all the children.
 
-Here, as a third parameter, you can also pass all include attributes but they will not get prefixed:
+Any of the included attributes can be passed as a third parameter here, but they will not be prefixed:
```

Comment:
```suggestion
Any of the included attributes can be passed as the third parameter here, but they will not be prefixed:
```

Reply:
Will not be prefixed?

Reply:
Oh sorry, I misread that... ignore my comment here.